src/util/logger.ts

```diff
@@ -0,0 +1,17 @@
+import { LIQUIBASE_LABEL } from 'constants/defaults';
```

Comment:
@RomanDeveloperAcc let's move this to `**/constants/tokens`
```suggestion
import { LIQUIBASE_LABEL } from 'constants/tokens';
```

We'll need a barrel export index.ts file in that directory as well :)